app/classifier/tasks/mobile-section.jsx

```diff
@@ -0,0 +1,145 @@
+    }
+  }
+
+  render() {
+    const nonShortCutTasks = Object.keys(this.props.workflow.tasks).filter((key) => { return this.props.workflow.tasks[key].type !== 'shortcut'; });
+    const shortcut = this.props.workflow.tasks[this.props.task.unlinkedTask];
+    const config = this.props.workflow.configuration;
+
+    // the following are criteria for the swipe app
+    const hasTwoAnswers = this.props.task.answers.length === 2;
+    const hasSingleTask = nonShortCutTasks.length === 1;
+    const questionNotTooLong = this.props.task.question.length < 200;
+    const notTooManyShortcuts = shortcut ? shortcut.answers.length <= 2 : true;
+    const isNotFlipbook = config ? config.multi_image_mode !== 'flipbook' : true;
+    const doesNotUseFeedback = this.props.task.feedback ? !this.props.task.feedback.enabled : true;
+
+    const swipeEligible = hasTwoAnswers && hasSingleTask && questionNotTooLong && notTooManyShortcuts && isNotFlipbook && doesNotUseFeedback;
+
+    const disabledStyle = {
+      opacity: 0.5,
+      pointerEvents: 'none'
+    };
+
+    const doesntMeet =
+      <i className="fa fa-times" style={{ color: 'red' }} aria-hidden="true" />;
+
+    const meets =
+      <i className="fa fa-check" style={{ color: 'green' }} aria-hidden="true" />;
+
+    const ineligibleText = (
+      <small className="form-help">
+        Sorry, but the mobile app will not currently work for this workflow.
+        <br />
+        The following are the requirements for the swipe workflow.
+        <ul>
+          <li>Has one Task { hasSingleTask ? meets : doesntMeet }</li>
+          <li>Has two available answers (e.g. Yes/No) { hasTwoAnswers ? meets : doesntMeet }</li>
+          <li>Question has less than 200 characters { questionNotTooLong ? meets : doesntMeet }</li>
+          <li>Has less than three shortcuts { notTooManyShortcuts ? meets : doesntMeet }</li>
+          <li>Cannot be a flipbook { isNotFlipbook ? meets : doesntMeet }</li>
+          <li>Cannot provide feedback { doesNotUseFeedback ? meets : doesntMeet }</li>
+        </ul>
+      </small>
+    );
+
+    const eligibleText = (
+      <small className="form-help">
+        Check this box if you think your question fits in this way.  If you have a Yes/No question, we recommend Yes is the first option listed so that it appears on the right.
+        <br />
+        <br />
+      </small>
+    );
+
+    const mobileHelp = 'Mobile app:  Check this box if you would like this workflow available in the mobile app';
+    const swipeEnabledChecked = !!this.props.workflow.configuration.swipe_enabled && swipeEligible;
+    return (
+      <div>
+        <span className="form-label">Mobile App</span>
+        <br />
+        <div className="workflow-mobile-form">
+          <div className="workflow-mobile-form-left-panel">
+            <label
+              className="pill-button"
+              htmlFor="swipe_enabled"
+              title={mobileHelp}
```

Comment:
This will not be visible if the style of the element sets `pointerEvents: none`. Might be best to disable this another way.